examples/output_voltage_measurement/measurement.py

```diff
@@ -182,7 +182,7 @@ def _get_session_info_for_pin(
 
 def _wait_for_source_complete_event(
     measurement_service: nims.MeasurementService,
-    channels: nidcpower._SessionBase,
+    channels: nidcpower.Session,
```

Comment:
This is incorrect. `session.channels[x]` returns a `_SessionBase`, not a `Session`. 

It only passes type checking because nimi-python doesn't have type hints and `mypy --ignore-missing-imports` treats missing modules/classes/methods as `Any`.

A better fix is to add `from __future__ import annotations` so that Python doesn't try to pass type hints at run time. However, I don't understand why this ever failed.

Reply:
Strange. I wonder why _SessionBase didn't show up when I searched the repo. What you linked to above wasn't the main branch, but switching to the main branch it is there too. 

I'll try you suggested change and see if I can run the measurement.

Reply:
Also, using the wrong type will cause editors like VS Code and PyCharm to offer the wrong auto-complete suggestions. For example, the `self_test` method is defined by `Session` but not `_SessionBase`, and using the wrong type will allow your editor to auto-complete a call to `channels.self_test`.

Reply:
Adding `from __future__ import annotations` fixes the issue.